deploy/scripts/src/ipv-core/test.ts

```diff
@@ -462,12 +538,13 @@ export function coreScenario1 (): void {
   sleep(Math.random() * 3)
 
   group(
-    'B01_Core_09_KBVDataContinue POST',
+    'B01_Core_10_KBVDataContinue POST',
     function () {
       const startTime1 = Date.now()
       res = http.post(
         kbvStubURL,
         {
+          sort: 'Kenneth Decerqueira (Valid Experian) KBV',
```

Comment:
```suggestion
          sort: user.kbv,
```
Suggestion: Paramaterise these values

Reply:
Done